CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+# 0.12.0
+
+## Allow access to system site packages
+
+* Use `dmenv --system-site-packages install`  to create a virtualenv that has access to the system's site packages, similarly with `dmenv --system-site-packages` for `dmenv lock`.
```

Comment:
Huge release-blocker: There's two spaces after the `, before "to create" !

Also, I think that should be `similarly to` instead of `with`?